app.rb

```diff
@@ -0,0 +1,111 @@
+require './book'
+require './capitalize_decorator'
+require './classroom'
+require './decorator'
+require './nameable'
+require './person'
+require './rental'
+require './student'
+require './teacher'
+require './trimmer_decorator'
+
+class App
+  attr_reader :people, :books
+
+  def initialize
+    @people = []
+    @books = []
+  end
+
+  def list_people
+    if @people.empty?
+      puts
+      puts 'No people in system.'
+    else
+      @people.each { |person| puts "[#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}" }
+    end
+  end
+
+  def list_books
+    if @books.empty?
+      puts
+      puts 'No books in system.'
+    else
+      @books.each { |book| puts "Title: \"#{book.title}\" , Author: #{book.author}" }
+    end
+  end
+
+  def create_teacher
+    print 'Age: '
+    age = gets.chomp
+    print 'Name: '
+    name = gets.chomp
+    print 'Specialization: '
+    specialization = gets.chomp
+    new_person = Teacher.new(age, specialization, name)
+    @people << new_person
+    puts 'Person created successfully'
+  end
```

Comment:
- [ ] I recognised that when I am creating a `teacher` In place of age there is specialization please address that
![teacher](https://user-images.githubusercontent.com/44745855/169669892-1836b578-fa77-4071-ab7f-ee60a676ef82.png)

Reply:
Done!